src/atomizer.js

```diff
@@ -678,11 +684,6 @@ Atomizer.prototype.getCss = function (config/*:AtomizerConfig*/, options/*:CSSOp
         }
     });
 
-    if (options.namespace) {
-        var jssNew = {};
-        jssNew[options.namespace] = jss;
-        jss = jssNew;
-    }
     if (options.helpersNS) {
```

Comment:
@thierryk Currently namespaces on helpers is optional, with the default being disabled.  What I'm more concerned by is, if you do specify a namespace for helpers, shouldn't we skip the namespace if a contextual selector is used, just like we do for atomic classes?